tap_dynamics/client.py

```diff
@@ -44,25 +45,37 @@ def retry_after_wait_gen():
 
 
 class DynamicsException(Exception):
-    pass
+    def __init__(self, message=None, response=None):
+        super().__init__(message)
+        self.message = message
+        self.response = response
 
 # pylint: disable=missing-class-docstring
 
 
 class DynamicsQuotaExceededException(DynamicsException):
-    pass
+    def __init__(self, message=None, response=None):
+        super().__init__(message)
+        self.message = message
+        self.response = response
```

Comment:
Not a big deal, but do we need message and response separately in these subclasses, or can we just pass them both through to the DynamicsException parent class?